_posts/2018-03-01-Rust-1.24.1.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: post
+title: "Announcing Rust 1.24.1"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.24.1. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.24.1 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.24.1][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1241-2018-03-01
+
+## What's in 1.24.1 stable
+
+As you know, we tend not to release point releases. In this case, we had a few regressions in 1.24.0.
+Each of them individually was not a "hair on fire" moment, but there's two reasons we decided to
+produce a point release. First, collectively, they were big enough to consider as a whole. Second, we
+want our attitude towards regressions to be "if we're not sure, default to reverting." Given that
+it was not clear that we *shouldn't* release a point release, we decided to err on the side of
```

Comment:
I had to read this sentence 3 times to get it straight in my head. Is there another way we could word this?

Reply:
Personally, I would nix this whole paragraph, and just say that "Several minor regressions were found which collectively merited a release."

Reply:
I actually am not fond of the phrasing starting at "First, collectively...". I wouldn't enumerate the *reasons to produce a point release* at all, in fact, because the main thing I think a typical reader will want enumerated is the *set of regressions*, and enumerating two lists is more confusing than enumerating one.

So I'd suggest replacing the entire paragraph with:

 > As you know, we avoid point releases: new features are released as part of the standard release cadence, and our release process and beta testing do a fairly good job of helping us avoid regressions in each new release. When regressions do occur, however, our philosophy is to err on the side of reverting. In this case, there were a few regressions in 1.24.0 that were collectively important enough to necessitate this release.

Reply:
Dang, I thought I refreshed the page before drafting my comment, but apparently I didn't. @aturon is correct that we can eliminate the paragraph entirely if we want.